toml/README.md

```diff
@@ -93,3 +95,17 @@ const tomlObject = parseFile("file.toml");
 const tomlString = 'foo.bar = "Deno"';
 const tomlObject22 = parse(tomlString);
 ```
+
+### Dump
+
+```ts
+import { dump } from "./parser.ts";
+const obj = {
+  bin: [
+    { name: "deno", path: "cli/main.rs" },
+    { name: "deno_core", path: "src/foo.rs" }
+  ],
+  nib: [{ name: "node", path: "not_found" }]
+};
+const tomlString = dump(obj);
```

Comment:
Rename to “stringing” ?

Reply:
Most of the time the name `dump` is used. For example:
- https://www.npmjs.com/package/js-yaml
- https://pyyaml.org/wiki/PyYAMLDocumentation
- https://pypi.org/project/toml/

Reply:
But i can rename if you want.

Reply:
Yes please rename.